live/dash/live_dash.py

```
@@ -489,25 +489,113 @@ def get_state():
-            return False
+            times = _win_process_times(int(pid))
+            if times is None:
+                return False
+            created, exited = times
+            if exited is not None:
+                # Queryable but finished -- a handle is still open somewhere.
+                return False
         else:
             os.kill(int(pid), 0)
-            return True
     except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)